zap/src/main/java/org/zaproxy/zap/spider/parser/SpiderHtmlParser.java

```diff
@@ -41,13 +44,21 @@ public class SpiderHtmlParser extends SpiderParser {
 
     /** The Constant URL_PATTERN defining the pattern for a meta url. */
     static final Pattern URL_PATTERN =
-            Pattern.compile("url\\s*=\\s*[\"']?([^;'\"]+)", Pattern.CASE_INSENSITIVE);
+            Pattern.compile(
+                    "(?:url\\s*=|report-uri)\\s*[\"']?([^;'\"]+)", Pattern.CASE_INSENSITIVE);
 
     private static final Pattern PLAIN_COMMENTS_URL_PATTERN =
             Pattern.compile(
-                    "(?:http(?:s?):)?//[^\\x00-\\x1f\"'\\s<>#()\\[\\]{}]+",
+                    "(?:http(?:s?):)?/?/[^\\x00-\\x1f\"'\\s<>#()\\[\\]{}]+",
```

Comment:
I don't think we should include everything that has a slash. This also matches invalid URLs e.g. `http:/example.com/`.

Reply:
Updated the regex and now it should ideally match all cases. Please check :)

Reply:
Introduce a new regex just for inline content, this is now matching `/a` in plain comments which didn't before.

Reply:
Done